src/main.rs

```diff
@@ -30,8 +30,9 @@ Spellcheck all your doc comments
 Usage:
     cargo-spellcheck [(-v...|-q)] check [--cfg=<cfg>] [--checkers=<checkers>] [[--recursive] -- <paths>... ]
     cargo-spellcheck [(-v...|-q)] fix [--cfg=<cfg>] [--checkers=<checkers>] [[--recursive] -- <paths>... ]
-    cargo-spellcheck [(-v...|-q)] config [--force] [--user] [--cfg=<cfg>]
-    cargo-spellcheck [(-v...|-q)] [--cfg=<cfg>] [(--fix|--interactive)] [--checkers=<checkers>] [[--recursive] -- <paths>... ]
+    cargo-spellcheck [(-v...|-q)] config (--user|--stdout|--cfg=<cfg>) [-f|--force]
```

Comment:
`[--force]` only makes sense for `--user` and `--cfg`, not for `--stdout`, but I guess that can be ignored

Reply:
You're right, but making it just optional for `--user` and `--cfg` makes it less readable so I also vote for ignoring